scripts/openshiftci-presubmit-e2e-tests.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# Do not show token in CI log
+set +x
+
+# show commands
+set -x
+
+# Set necessary env variables
+export PATH="$PATH:$(pwd)"
+KUBECONFIG=${KUBECONFIG:-$HOME/.kube/config}
+
+# Install Argo CD
+make install-argocd-openshift
+
+# Deploy GitOps Service components
+# Use Image built by CI, image name is provided as env variable 'CI_IMAGE' by CI
+echo "Using Image $CI_IMAGE"
+make install-all-k8s IMG=$CI_IMAGE
+
+# Port forwarding is needed by E2E to connect with DB
+kubectl port-forward --namespace gitops svc/gitops-postgresql-staging 5432:5432 &
```

Comment:
`make install-all-k8s` will also port-forward after installing everything. Do we still need to explicitly port-forward?